src/components/listing/FileLister.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div></div>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop } from 'vue-property-decorator'
+import { NoCache } from 'vue-class-decorator'
+import { MutationPayload } from 'vuex'
+import { getModule } from 'vuex-module-decorators'
+import ListModule from '../../modules/ListModule'
+import Web3Module from '../../modules/Web3Module'
+import MetaMaskModule from '../../modules/MetaMaskModule'
+import FfaListingsModule from '../../modules/FfaListingsModule'
+import Listing from '../../models/protocol/Listing'
+import FfaListing from '../../models/FfaListing'
+import { ProcessStatus } from '../../models/ProcessStatus'
+import { Errors, Labels, Messages } from '../../util/Constants'
+import Web3 from 'web3'
+
+const vuexModuleName = 'listModule'
+
+@Component
+export default class FileLister extends Vue {
+
+  public mounted(this: FileLister) {
+    this.$store.subscribe(this.vuexSubscriptions)
+  }
+
+  private vuexSubscriptions(mutation: MutationPayload, state: any) {
+    switch (mutation.type) {
```

Comment:
if there are more options coming thru here later and you are being future-friendly.... OK

if not kill this *switchception* and use good ol `if/else`

Reply:
should be moar coming.